### DISCUSSION TITLE
How to modify a wrong System Paths Directory

### DISCUSSION THREAD

**Discussion by jcuell2** [vladmandic/sdnext#596] [General]
How would I modify a wrong system path directory as whenever I launch webui.bat my available models goes to a wrong path (as I was trying to get multiple sources for models). 

My screen tells me this (ignoring the update):
No module 'xformers'. Proceeding without it.
Available models: C:\Users\Name\Downloads\Super_SD2\stable-diffusion-webui\models\Stable-diffusion
D:\Super Sd 0
Download the default model? (y/N) y
ControlNet v1.1.116
ControlNet v1.1.116
Loading theme: gradio/default
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Initializing middleware
Cannot run without a checkpoint
Use --ckpt <path-to-checkpoint> to force using existing checkpoint
Press any key to continue . . .

Specifically my problem is at  "Available models: C:\Users\Jose\Downloads\Super_SD2\stable-diffusion-webui\models\Stable-diffusion
D:\Super Sd 0", I've tried downloading the default model (y and n) with same result of just crashing. I've also done the same while removing the usb (D drive) with similar results. I've also tried to access the local URL but i'm unable to get a connection. I was hoping I would since I would just go back and modify my System Path. As asked before I was told I can add a subset folder and Vlad would automatically open it up, due to current issues I've removed said subset folder and got the same crash result. Is there a way to change System Path without opening the local URL? I've tried opening a couple of files in notepad++ while searching "ckpt" but I think asking here would be easier. Every time I open my Webui.bat I get this result and unable to load Local URL. 
I believe wrote this for my System Path for model:
C:\Users\Name\Downloads\Super_SD2\stable-diffusion-webui\models\Stable-diffusion --ckpt <D:\Super Sd>

**Comment by vladmandic** [comment]
path is written in `config.json` value `ckpt_dir`. you can edit the file or you can just delete it and app will create new one with default values on next startup.
